src/mozanalysis/experiment.py

```diff
@@ -798,32 +873,62 @@ def _build_enrollments_query_cirrus(
-            WHERE
-                event_category = 'normandy'
-                AND (event_method = 'exposure' OR event_method = 'expose')
-                AND submission_date
-                    BETWEEN '{time_limits.first_enrollment_date}' AND '{time_limits.last_enrollment_date}'
-                AND event_string_value = '{self.experiment_slug}'
-        ) e
-        ON re.client_id = e.client_id AND
-            re.branch = e.branch AND
-            e.submission_date >= re.enrollment_date
-        GROUP BY e.client_id, e.branch
-            """  # noqa: E501
+                e.client_id,
+                e.branch,
+                min(e.submission_date) AS exposure_date,
+                COUNT(e.submission_date) AS num_exposure_events
+            FROM raw_enrollments re
+            LEFT JOIN (
+                SELECT
+                    client_id,
+                    `mozfun.map.get_key`(event_map_values, 'branchSlug') AS branch,
+                    submission_date
+                FROM
+                    `moz-fx-data-shared-prod.telemetry.events`
+                WHERE
+                    event_category = 'normandy'
+                    AND (event_method = 'exposure' OR event_method = 'expose')
+                    AND submission_date
+                        BETWEEN '{time_limits.first_enrollment_date}' AND '{time_limits.last_enrollment_date}'
+                    AND event_string_value = '{self.experiment_slug}'
+            ) e
+            ON re.client_id = e.client_id AND
+                re.branch = e.branch AND
+                e.submission_date >= re.enrollment_date
+            GROUP BY e.client_id, e.branch
+                """  # noqa: E501
+        elif self.experimental_unit == ExperimentalUnit.GROUP:
+            return f"""
+            SELECT
+                e.profile_group_id,
+                e.branch,
+                min(e.submission_date) AS exposure_date,
+                COUNT(e.submission_date) AS num_exposure_events
+            FROM raw_enrollments re
+            LEFT JOIN (
+                SELECT
+                    profile_group_id,
+                    `mozfun.map.get_key`(event_map_values, 'branchSlug') AS branch,
+                    submission_date
+                FROM
+                    `moz-fx-data-shared-prod.telemetry.events`
+                WHERE
+                    event_category = 'normandy'
+                    AND (event_method = 'exposure' OR event_method = 'expose')
+                    AND submission_date
+                        BETWEEN '{time_limits.first_enrollment_date}' AND '{time_limits.last_enrollment_date}'
+                    AND event_string_value = '{self.experiment_slug}'
+            ) e
+            ON re.profile_group_id = e.profile_group_id AND
+                re.branch = e.branch AND
+                e.submission_date >= re.enrollment_date
+            GROUP BY e.profile_group_id, e.branch
+                """  # noqa: E501
+        else:
```

Comment:
Same comment as above around not having 2 copies of nearly the same query.

Reply:
Initially had them separate because they could vary... they ended up initially varying much less than than expected. Why don't we prohibit downsampling for group level experiments until the group level sampling is resolved

Reply:
Even if we did that, which is reasonable, we could still use string formatting+variables to do it I think...though that does make the queries more distinct. I just personally never like seeing more than a few line SQL queries like this copied & pasted, especially when they already use a few string format vars, but I leave it up to you.

Reply:
Changed.